integration/kubernetes/confidential/agent_image.bats

```diff
@@ -49,6 +49,20 @@ setup() {
 	clear_kernel_params
 	add_kernel_params \
 		"agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"
+	if [ -n "$https_proxy" ]; then
```

Comment:
I remember the first time @arronwy introduced the handler for https proxying, the url could be passed via `HTTPS_PROXY` (uppercase) or `https_proxy` (lowercase). See https://github.com/kata-containers/tests/blob/CCv0/integration/containerd/confidential/agent_api.bats#L22

I am saying that just to be sure you don't need such as handling here as well.

Apart from that, I think we should refactor the code to avoid those boilerplate. In a future PR, I mean.

Reply:
Done!